client/lib/purchases/index.js

```diff
@@ -74,6 +78,14 @@ function getName( purchase ) {
 	return purchase.productName;
 }
 
+function getDescription( purchase ) {
+	const jetpackProductsDescriptions = getJetpackProductsDescriptions();
+	if ( jetpackProductsDescriptions[ purchase.productSlug ] ) {
+		return jetpackProductsDescriptions[ purchase.productSlug ];
+	}
+	return getName( purchase );
```

Comment:
I don't think this is a great fallback behavior, since the product name doesn't really make sense as a subscription.

I would suggest just doing `return '';` here.

Reply:
Great point, David — fixing that right away.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/40774/commits/4be9bc7f08dd9f40eba703741bc7a30d4acf3592

Ended up returning `null` to make it more consistent with similar functions on the same file (e.g.: `getPartnerName`).